src/main/java/TaskList.java

```diff
@@ -0,0 +1,70 @@
+import java.util.ArrayList;
+
+public class TaskList{
+
+    public int size = 0;
+    protected ArrayList<Task> list = new ArrayList<Task>();
+    public TaskList(){}
+
+    public void addToDo(String description){
+        ToDo toDo = new ToDo(description);
+        list.add(toDo);
+        this.size++;
+    }
+
+    public void addDeadline(String description, String dueDate){
+        Deadline deadline = new Deadline(description, dueDate);
+        list.add(deadline);
+        this.size++;
+    }
+
+    public void addEvent(String description, String dateTime){
+        Event event = new Event(description, dateTime);
+        list.add(event);
+        this.size++;
+    }
+
+    public Task getTask(int index){
+        return (Task)list.get(index);
+    }
+
+    public void printList(){
+        for (int i=0; i<size; i++){
+            Task tempTask = this.getTask(i);
+            String taskType = tempTask.getTaskType();
+            if (taskType == "T"){
```

Comment:
Magic String, maybe set "T" as a constant TASK = "T" so its easier for readers to know that T corresponds to task :)